lib/commands/cookies.js

```diff
@@ -1,67 +0,0 @@
-import _ from 'lodash';
-import { errors } from 'appium/driver';
-
-
-let commands = {}, helpers = {}, extensions = {};
-
-commands.getCookies = async function getCookies () {
```

Comment:
I wonder if this cookies.js's ones, via inspector protocol, are better than in web.js, which is JavaScript.
https://github.com/appium/appium-remote-debugger/blob/e2e34bfbb21fee93a23560a70592f8e1c1e1da91/lib/remote-debugger.js#L211

This cookies.js exists for long, while the web.js's `getCookies` for example was moved from appium-ios-driver. The previous stable release Appium 1.22.3 only has this cookies.js's one, so I guess potentially these cookies's ones changed the behavior (and not tested widely as beta)

Reply:
So... maybe we should keep this cookies.js, and remove the web.js' same naming methods

Reply:
Confirmed that the latest xcuitest called lib/commands/cookies.js side, so we should keep the cookies.js and remove the same names in web.js.

e.g.
```
2023-01-24 13:40:50:525 - [HTTP] [HTTP] --> DELETE /wd/hub/session/b8328303-fe24-423d-adba-214d6d497b43/cookie
2023-01-24 13:40:50:525 - [HTTP] [HTTP] {}
2023-01-24 13:40:50:525 - [debug] [XCUITestDriver@61e9 (b8328303)] [XCUITestDriver@61e9 (b8328303)] Calling AppiumDriver.deleteCookies() with args: ["b8328303-fe24-423d-adba-214d6d497b43"]
2023-01-24 13:40:50:525 - [debug] [XCUITestDriver@61e9 (b8328303)] [XCUITestDriver@61e9 (b8328303)] Executing command 'deleteCookies'
2023-01-24 13:40:50:525 - [debug] [RemoteDebugger] [RemoteDebugger] Getting cookies
2023-01-24 13:40:50:526 - [debug] [RemoteDebugger] [RemoteDebugger] Sending '_rpc_forwardSocketData:' message to app 'PID:1098', page '1', target 'page-8' (id: 40): 'Page.getCookies'
2023-01-24 13:40:50:529 - [debug] [RemoteDebugger] [RemoteDebugger] Received data response from send (id: 40): '{"cookies":[]}'
2023-01-24 13:40:50:529 - [debug] [RemoteDebugger] [RemoteDebugger] Sending to Web Inspector took 4ms
2023-01-24 13:40:50:529 - [debug] [XCUITestDriver@61e9 (b8328303)] [XCUITestDriver@61e9 (b8328303)] Responding to client with driver.deleteCookies() result: true
2023-01-24 13:40:50:530 - [HTTP] [HTTP] <-- DELETE /wd/hub/session/b8328303-fe24-423d-adba-214d6d497b43/cookie 200 5 ms - 14
```
https://github.com/appium/appium-remote-debugger/blob/e2e34bfbb21fee93a23560a70592f8e1c1e1da91/lib/remote-debugger.js#L213